packages/select/docs/FilterMultiSelect.stories.tsx

```diff
@@ -69,6 +70,63 @@ export const DefaultKaizenSiteDemo = args => {
 
 DefaultKaizenSiteDemo.storyName = "Default (Kaizen Site Demo)"
 
+export const TruncatedLabels = args => {
```

Comment:
Do you want this story to have controls? If so, you would want to spread the `args` into the story somewhere. Otherwise, disable the controls and change `args` to `()`.

Bonus points if you type the story too with either `Story` or `ComponentStory` :)